coalib/bearlib/aspects/decorators.py

```diff
@@ -0,0 +1,37 @@
+
+from functools import wraps
+
+from .taste import Taste
+from .meta import aspectclass
+
+
+def map_setting_to_aspect(**aspectable_setting):
+    """
+    Override function arguments value with coala's aspect and taste.
```

Comment:
'Override' ?

What happens if a setting is explicitly in .coafile, and is in an aspect? The aspect overrides the explicit setting?  I hope not.

Reply:
Um yes it is. It will **always** override explicit or default setting with aspect/taste setting if coala is run under aspect, except for those setting that doesn't mapped here (could be used by a very specific linter setting like `pep_ignore`) . Do we want different behavior here?

Reply:
Priority: old-setting default -> explicit old-setting -> aspect taste default -> explicit aspect taste

Reply:
explicit user choices should never be overriden automatically.
I think it should be:

explicit setting -> explicit aspect taste (if aspects  activated in Section) -> aspect taste default (if aspects activated in Section) -> setting default

(please stop calling them 'old-setting' - `Setting` is still a core component, and people will still keep using them at least until aspects are perfect for *everyones* needs, which wont be soon, and even then a Setting may still be used internally by the `run` method, until we also deprecate the `run` method. )

Reply:
Okay I agree, will change it.

Reply:
Then this function docstring needs to change.